api/src/main/resources/init-fluffy_owner.sql

```diff
@@ -0,0 +1,111 @@
+--
+-- Name: fluffy-broccoli; Type: SCHEMA; Schema: -; Owner: fluffy_owner
```

Comment:
Ik mis volgens mij even iets. Waar is dit voor i.c.m. jouw applicatie?

Reply:
Hahaha nou ik heb de shell van de database opgezet met heb geen tijd gehad om hem verder op te bouwen, hier moesten de false positives ed in worden opgeslagen

Reply:
Dit is dan gewoon een sql-bestand wat je nog van een ander project had? Anders snap ik namelijk niet de overeenkomsten tussen de tabellen en kolommen en jouw project :)

Reply:
Ja klopt, ik heb drie tabellen laten staan om ze later te vervangen om iig te kunnen zien of de database echt draaiende was ed.